### PR TITLE
Adding button to copy host name.

### PR DESCRIPTION
**Pull request by TheLat**

### CODE REVIEW

**Comment by yongwen** [comment]
can you add the description of the PR and if possible attach the image of the propose UI changes. Thanks

File: deploy-board/deploy_board/templates/hosts/host_details.html [hunk 1]
```diff
@@ -42,7 +42,7 @@
 
 {% if agent_wrappers %}
     <div class="panel panel-default">
-    {% include "panel_heading.tmpl" with panel_title="Agent Details" panel_body_id="agentDetailsId" direction="down" %}
+    {% include "panel_heading.tmpl" with panel_title='Agent Details</a> <img id="copyHostName" style="max-width:20px; margin-left: 3px; margin-bottom: 3px;" src="/static/img/copy.png" title="Copy host name to clipboard."><a>' panel_body_id="agentDetailsId" direction="down" %}
```
Comment (yongwen) [inline deploy-board/deploy_board/templates/hosts/host_details.html]: why  </a> at the beginning

File: deploy-board/deploy_board/templates/hosts/host_details.html [hunk 2]
```diff
@@ -56,7 +56,7 @@
 {% endif %}
 
 <div class="panel panel-default">
-{% include "panel_heading.tmpl" with panel_title="Host Details" panel_body_id="hostDetailId" direction="down" %}
+{% include "panel_heading.tmpl" with panel_title='Host Details</a> <img id="copyHostName" style="max-width:20px; margin-left: 3px; margin-bottom: 3px;" src="/static/img/copy.png" title="Copy host name to clipboard."><a>' panel_body_id="hostDetailId" direction="down" %}
```
Comment (yongwen) [inline deploy-board/deploy_board/templates/hosts/host_details.html]: same comment as above

**Comment by TheLat** [comment]
Adds an image based on github's copy to clipboard button.

**Pull request merged**